autobuild/mac/artifacts/autobuild_mac_1_prepare.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -e
+
+# autobuild_1_prepare: set up environment, install Qt & dependencies
+
+###################
+###  PROCEDURE  ###
+###################
+
+echo "Install dependencies..."
+#brew install qt5
```

Comment:
(non-blocking) lose the comments - they'll be in git history should we ever want them.

Reply:
done